### PR TITLE
tried to fix the issue 

### PR DESCRIPTION
**Pull request by Sumitkaranjkar**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -93,7 +93,7 @@ <h1 class="name">Shubham More</h1>
           </div>
         </div>
 
-        <div class="block">
+         <div class="block">
           <div>
             <img class="student-img" src=https://avatars.githubusercontent.com/u/98700773?s=400&u=5b02ac4c23f43b4bcc7129f4da5a659c91515ad0&v=4>
```
Comment (SahilK-027) [inline index.html]: Hey, src should go in quotes.
```html
<img class="student-img" src="https://avatars.githubusercontent.com/u/98700773?s=400&u=5b02ac4c23f43b4bcc7129f4da5a659c91515ad0&v=4">
```

**Review by SahilK-027 — CHANGES_REQUESTED** [review]
Add the link in double quotes and your code will work fine.

**Comment by SahilK-027** [comment]
And you don't need to close PR then make changes and open new PR again. Just do the suggested changes in your fork, give a proper commit message, and sync the fork, that's it. Your updations will be automatically added to the PR.

**Comment by Sumitkaranjkar** [comment]
now you can merge thanks

**Review by SahilK-027 — APPROVED** [review]

**Comment by SahilK-027** [comment]
Thanks for the PR

**Pull request merged**